about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <title>Tammy Fisher's Resume</title>
+
+    <link rel="stylesheet" href="css/index.css" />
+    <meta name="viewport" content="width=divice-width, initial-scale=1.0" />
+
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body class="container">
```

Comment:
I'm just going to have to say it.  I'm going to declare myself wrong.  I think it's better to just have a plain body tag and then have a div with a container class inside it.  That way, you can set the container to have a 80% width.